.circleci/config.yml

```diff
@@ -195,6 +195,7 @@ jobs:
       - setup
       - run: RAILS_ENV=test bundle exec rails db:create
       - run: RAILS_ENV=test bundle exec rails db:test:prepare
+      - run: yarn install
```

Comment:
I'm not really sure why this was necessary, because it should have been run in the build step, but it did fix the issue when I ssh'd in to the circleCI job, and just now when the full circleCI suite ran